tests/posix/fsstress.c

```diff
@@ -608,7 +608,10 @@ int FsstressStart(
     make_freq_table();
 
     while ((loopcntr <= loops) || (loops == 0)) {
-        RedSNPrintf(buf, sizeof(buf), "fss%x", getpid());
+        if (RedSNPrintf(buf, sizeof(buf), "fss%x", getpid()) < 0) {
+            RedPrintf("%s: error building name\n", __func__);
```

Comment:
We support older compilers (like Visual Studio 2010) that don't implement C99 features, such as the `__func__` macro.  Instead, the function name can be hard-coded in the string.